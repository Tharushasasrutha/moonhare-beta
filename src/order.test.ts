import {layerOrder, pluginOrder, variantOrder} from './order'

describe('Order', () => {
    test('layerOrder', () => {
        expect(layerOrder).toMatchInlineSnapshot(`
Array [
  "base",
  "components",
  "shortcuts",
  "utilities",
]
`)
    })
    test('pluginOrder', () => {
        expect(pluginOrder).toMatchSnapshot()
    })
    test('variantOrder', () => {
        expect(variantOrder).toMatchSnapshot()
    })
})
