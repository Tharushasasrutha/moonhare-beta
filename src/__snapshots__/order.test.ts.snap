// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order pluginOrder 1`] = `
Object {
  "100": "container",
  "1000": "backgroundClip",
  "10000": "gridColumn",
  "10100": "gridColumnStart",
  "10200": "gridColumnEnd",
  "10300": "gridTemplateRows",
  "10400": "gridAutoRows",
  "10500": "gridRow",
  "10600": "gridRowStart",
  "10700": "gridRowEnd",
  "10800": "transform",
  "10900": "transformOrigin",
  "1100": "backgroundColor",
  "11000": "scale",
  "11100": "rotate",
  "11200": "translate",
  "11300": "skew",
  "11350": "perspective",
  "11360": "perspectiveOrigin",
  "11400": "transitionProperty",
  "11500": "transitionTimingFunction",
  "11600": "transitionDuration",
  "11700": "transitionDelay",
  "11800": "keyframes",
  "11900": "animation",
  "11950": "imageRendering",
  "1200": "backgroundImage",
  "12000": "mixBlendMode",
  "12100": "backgroundBlendMode",
  "12200": "filter",
  "12300": "blur",
  "12400": "brightness",
  "12500": "contrast",
  "12600": "dropShadow",
  "12700": "grayscale",
  "12800": "hueRotate",
  "12900": "invert",
  "1300": "gradientColorStops",
  "13000": "saturate",
  "13100": "sepia",
  "13200": "backdropFilter",
  "13300": "backdropBlur",
  "13400": "backdropBrightness",
  "13500": "backdropContrast",
  "13600": "backdropGrayscale",
  "13700": "backdropHueRotate",
  "13800": "backdropInvert",
  "13900": "backdropOpacity",
  "1400": "backgroundOpacity",
  "14000": "backdropSaturate",
  "14100": "backdropSepia",
  "1500": "backgroundPosition",
  "1600": "backgroundRepeat",
  "1700": "backgroundSize",
  "1750": "backgroundOrigin",
  "1800": "borderCollapse",
  "1900": "borderColor",
  "200": "space",
  "2000": "borderOpacity",
  "2100": "borderRadius",
  "2200": "borderStyle",
  "2300": "borderWidth",
  "2350": "boxDecorationBreak",
  "2400": "boxSizing",
  "2500": "cursor",
  "2550": "captionSide",
  "2560": "emptyCells",
  "2600": "display",
  "2700": "flexDirection",
  "2800": "flexWrap",
  "2900": "placeItems",
  "300": "divideWidth",
  "3000": "placeContent",
  "3100": "placeSelf",
  "3200": "alignItems",
  "3300": "alignContent",
  "3400": "alignSelf",
  "3500": "justifyItems",
  "3600": "justifyContent",
  "3700": "justifySelf",
  "3800": "flex",
  "3900": "flexGrow",
  "400": "divideColor",
  "4000": "flexShrink",
  "4100": "order",
  "4200": "float",
  "4300": "clear",
  "4400": "fontFamily",
  "4500": "fontWeight",
  "4600": "height",
  "4700": "fontSize",
  "4800": "lineHeight",
  "4900": "listStylePosition",
  "500": "divideStyle",
  "5000": "listStyleType",
  "5100": "margin",
  "5200": "maxHeight",
  "5300": "maxWidth",
  "5400": "minHeight",
  "5500": "minWidth",
  "5600": "objectFit",
  "5700": "objectPosition",
  "5800": "opacity",
  "5900": "outline",
  "600": "divideOpacity",
  "6000": "overflow",
  "6100": "overscrollBehavior",
  "6200": "padding",
  "6300": "placeholderColor",
  "6400": "placeholderOpacity",
  "6450": "caretColor",
  "6460": "caretOpacity",
  "6470": "tabSize",
  "6500": "pointerEvents",
  "6600": "position",
  "6700": "inset",
  "6800": "resize",
  "6900": "boxShadow",
  "6950": "boxShadowColor",
  "700": "accessibility",
  "7000": "ringWidth",
  "7100": "ringOffsetColor",
  "7200": "ringOffsetWidth",
  "7300": "ringColor",
  "7400": "ringOpacity",
  "7500": "fill",
  "7600": "stroke",
  "7700": "strokeWidth",
  "7750": "strokeDashArray",
  "7760": "strokeDashOffset",
  "7800": "tableLayout",
  "7900": "textAlign",
  "800": "appearance",
  "8000": "textColor",
  "8100": "textOpacity",
  "8200": "textOverflow",
  "8250": "textShadow",
  "8300": "fontStyle",
  "8400": "textTransform",
  "8450": "textDecorationStyle",
  "8455": "textDecorationLength",
  "8460": "textDecorationColor",
  "8470": "textDecorationOpacity",
  "8480": "textDecorationOffset",
  "8500": "textDecoration",
  "8550": "textIndent",
  "8560": "textStrokeColor",
  "8570": "textStrokeWidth",
  "8580": "content",
  "8600": "fontSmoothing",
  "8700": "fontVariantNumeric",
  "8800": "letterSpacing",
  "8900": "userSelect",
  "900": "backgroundAttachment",
  "9000": "verticalAlign",
  "9100": "visibility",
  "9150": "backfaceVisibility",
  "9200": "whitespace",
  "9300": "wordBreak",
  "9340": "writingMode",
  "9350": "hyphens",
  "9400": "width",
  "9500": "zIndex",
  "9550": "isolation",
  "9600": "gap",
  "9700": "gridAutoFlow",
  "9800": "gridTemplateColumns",
  "9900": "gridAutoColumns",
  "accessibility": 700,
  "alignContent": 3300,
  "alignItems": 3200,
  "alignSelf": 3400,
  "animation": 11900,
  "appearance": 800,
  "backdropBlur": 13300,
  "backdropBrightness": 13400,
  "backdropContrast": 13500,
  "backdropFilter": 13200,
  "backdropGrayscale": 13600,
  "backdropHueRotate": 13700,
  "backdropInvert": 13800,
  "backdropOpacity": 13900,
  "backdropSaturate": 14000,
  "backdropSepia": 14100,
  "backfaceVisibility": 9150,
  "backgroundAttachment": 900,
  "backgroundBlendMode": 12100,
  "backgroundClip": 1000,
  "backgroundColor": 1100,
  "backgroundImage": 1200,
  "backgroundOpacity": 1400,
  "backgroundOrigin": 1750,
  "backgroundPosition": 1500,
  "backgroundRepeat": 1600,
  "backgroundSize": 1700,
  "blur": 12300,
  "borderCollapse": 1800,
  "borderColor": 1900,
  "borderOpacity": 2000,
  "borderRadius": 2100,
  "borderStyle": 2200,
  "borderWidth": 2300,
  "boxDecorationBreak": 2350,
  "boxShadow": 6900,
  "boxShadowColor": 6950,
  "boxSizing": 2400,
  "brightness": 12400,
  "captionSide": 2550,
  "caretColor": 6450,
  "caretOpacity": 6460,
  "clear": 4300,
  "container": 100,
  "content": 8580,
  "contrast": 12500,
  "cursor": 2500,
  "display": 2600,
  "divideColor": 400,
  "divideOpacity": 600,
  "divideStyle": 500,
  "divideWidth": 300,
  "dropShadow": 12600,
  "emptyCells": 2560,
  "fill": 7500,
  "filter": 12200,
  "flex": 3800,
  "flexDirection": 2700,
  "flexGrow": 3900,
  "flexShrink": 4000,
  "flexWrap": 2800,
  "float": 4200,
  "fontFamily": 4400,
  "fontSize": 4700,
  "fontSmoothing": 8600,
  "fontStyle": 8300,
  "fontVariantNumeric": 8700,
  "fontWeight": 4500,
  "gap": 9600,
  "gradientColorStops": 1300,
  "grayscale": 12700,
  "gridAutoColumns": 9900,
  "gridAutoFlow": 9700,
  "gridAutoRows": 10400,
  "gridColumn": 10000,
  "gridColumnEnd": 10200,
  "gridColumnStart": 10100,
  "gridRow": 10500,
  "gridRowEnd": 10700,
  "gridRowStart": 10600,
  "gridTemplateColumns": 9800,
  "gridTemplateRows": 10300,
  "height": 4600,
  "hueRotate": 12800,
  "hyphens": 9350,
  "imageRendering": 11950,
  "inset": 6700,
  "invert": 12900,
  "isolation": 9550,
  "justifyContent": 3600,
  "justifyItems": 3500,
  "justifySelf": 3700,
  "keyframes": 11800,
  "letterSpacing": 8800,
  "lineHeight": 4800,
  "listStylePosition": 4900,
  "listStyleType": 5000,
  "margin": 5100,
  "maxHeight": 5200,
  "maxWidth": 5300,
  "minHeight": 5400,
  "minWidth": 5500,
  "mixBlendMode": 12000,
  "objectFit": 5600,
  "objectPosition": 5700,
  "opacity": 5800,
  "order": 4100,
  "outline": 5900,
  "overflow": 6000,
  "overscrollBehavior": 6100,
  "padding": 6200,
  "perspective": 11350,
  "perspectiveOrigin": 11360,
  "placeContent": 3000,
  "placeItems": 2900,
  "placeSelf": 3100,
  "placeholderColor": 6300,
  "placeholderOpacity": 6400,
  "pointerEvents": 6500,
  "position": 6600,
  "resize": 6800,
  "ringColor": 7300,
  "ringOffsetColor": 7100,
  "ringOffsetWidth": 7200,
  "ringOpacity": 7400,
  "ringWidth": 7000,
  "rotate": 11100,
  "saturate": 13000,
  "scale": 11000,
  "sepia": 13100,
  "skew": 11300,
  "space": 200,
  "stroke": 7600,
  "strokeDashArray": 7750,
  "strokeDashOffset": 7760,
  "strokeWidth": 7700,
  "tabSize": 6470,
  "tableLayout": 7800,
  "textAlign": 7900,
  "textColor": 8000,
  "textDecoration": 8500,
  "textDecorationColor": 8460,
  "textDecorationLength": 8455,
  "textDecorationOffset": 8480,
  "textDecorationOpacity": 8470,
  "textDecorationStyle": 8450,
  "textIndent": 8550,
  "textOpacity": 8100,
  "textOverflow": 8200,
  "textShadow": 8250,
  "textStrokeColor": 8560,
  "textStrokeWidth": 8570,
  "textTransform": 8400,
  "transform": 10800,
  "transformOrigin": 10900,
  "transitionDelay": 11700,
  "transitionDuration": 11600,
  "transitionProperty": 11400,
  "transitionTimingFunction": 11500,
  "translate": 11200,
  "userSelect": 8900,
  "verticalAlign": 9000,
  "visibility": 9100,
  "whitespace": 9200,
  "width": 9400,
  "wordBreak": 9300,
  "writingMode": 9340,
  "zIndex": 9500,
}
`;

exports[`Order variantOrder 1`] = `
Array [
  "hover",
  "focus",
  "active",
  "visited",
  "link",
  "target",
  "focus-visible",
  "focus-within",
  "checked",
  "not-checked",
  "default",
  "disabled",
  "enabled",
  "indeterminate",
  "invalid",
  "valid",
  "optional",
  "required",
  "placeholder-shown",
  "read-only",
  "read-write",
  "not-disabled",
  "first-of-type",
  "not-first-of-type",
  "last-of-type",
  "not-last-of-type",
  "first",
  "last",
  "not-first",
  "not-last",
  "only-child",
  "not-only-child",
  "only-of-type",
  "not-only-of-type",
  "even",
  "odd",
  "even-of-type",
  "odd-of-type",
  "root",
  "empty",
  "before",
  "after",
  "first-letter",
  "first-line",
  "selection",
  "svg",
  "all",
  "children",
  "siblings",
  "sibling",
  "ltr",
  "rtl",
  "group-hover",
  "group-focus",
  "group-active",
  "group-visited",
  "motion-safe",
  "motion-reduce",
]
`;
